services/blockchain-app-registry/config.js

```diff
@@ -63,7 +63,7 @@ config.gitHub = {
 	get appRegistryRepoName() { return this.appRegistryRepo.split('/').pop(); },
 };
 
-config.dataDir = `${process.cwd()}/data`;
+config.dataDir = `${__dirname}/data`;
```

Comment:
This change is incorrect. `process.cwd()` ensures that the `data` directory is created at the intended place, viz. the home directory of the microservice.

If using `__dirname`, please adjust it to be created at the right place using parent directory notations. (Eg: `${__dirname}/../../../data`)

Reply:
`${__dirname}/../../../data` was just an example.

Reply:
`${__dirname}/data` currently works because the config file exists in the application's root directory. So, it should be fine for now.